subxt/src/client.rs

```diff
@@ -179,6 +182,18 @@ impl<T: Config> Client<T> {
     pub fn to_runtime_api<R: From<Self>>(self) -> R {
         self.into()
     }
+
+    /// Returns a snapshot of the client Runtime Version.
+    pub fn runtime_version(&self) -> RuntimeVersion {
+        let runtime = self.runtime_version.read().unwrap();
+        runtime.clone()
+    }
+
+    /// Set the given Runtime Version on the client.
+    pub fn set_runtime_version(&self, runtime: RuntimeVersion) {
+        let mut actual = self.runtime_version.write().unwrap();
```

Comment:
One other option would be:
```rust
if actual.spec_version < runtime.spec_version {
    *actual = runtime;
}
```

And therefore eliminate the need for cloning the runtime whenever inspecting the code.